lib/tasks/works.rake

```diff
@@ -82,4 +82,16 @@ namespace :works do
       puts "\t#{file_name}"
     end
   end
+
+  # Use this to figure out what works don't have the proper DataCite attributes
+  # (useful when troubleshooting bad data already in the system)
+  desc "Shows what works are valid for DataCite submission."
+  task :validate_datacite_attributes, [:verbose] => :environment do |_, args|
+    Work.all.each do |work|
+      _attributes = work.send(:doi_attribute_xml)
```

Comment:
I believe the code below would trigger the same error.  I'm unsure why the doi_attribute_xml method exists.
```suggestion
      _attributes = work.resource.to_xml
```